api/src/client/ApiFormationHelpers.ts

```diff
@@ -42,6 +42,26 @@ export const makePostBody = (
     return currentBusiness.profileData.legalStructureId as FormationLegalType;
   };
 
+  const getMainBusinessAddress = (): ApiLocation | undefined => {
+    const isStartingBusinessPartiallyEmpty =
+      currentBusiness.profileData.businessPersona === "STARTING" &&
```

Comment:
use `isStartingBusiness` instead of `currentBusiness.profileData.businessPersona === "STARTING"`